torch/csrc/jit/passes/quantization.cpp

```diff
@@ -454,11 +457,6 @@ void InsertObserversHelper::insertObservers(
     Block* b = blocks_to_visit.top();
     blocks_to_visit.pop();
     for (Node* n : b->nodes()) {
-      // Skip observer nodes
-      if (observer_for_input.count(n) != 0) {
-        continue;
-      }
```

Comment:
Nit: It sill might be worth it not ti add such nodes to `values_to_observe`.

Reply:
would this be a bit redundant though, maybe I can remove the insertObserverFor then?